### ISSUE TITLE
Masking issue in AttentivePoolingModule

### ISSUE THREAD

**Issue by kpeverson** [s3prl/s3prl#579]
I believe there is an issue with how the attention mask is working in the [AttentivePoolingModule](https://github.com/s3prl/s3prl/blob/ec8064b5889f81ca460fbe2c094ce576a6f120b7/s3prl/downstream/model.py#L123). `att_mask` is `True` in valid frames and `False` in padded frames, from [line 94](https://github.com/s3prl/s3prl/blob/ec8064b5889f81ca460fbe2c094ce576a6f120b7/s3prl/downstream/model.py#L94). When this is added to `att_logits` in line 123, it simply adds `1` to valid frames and `0` to padded frames, biasing the attention weights towards the valid frames but not completely ignoring the padded frames. Unless there is a reason for this, I believe the correct approach should be:

```
att_logits = att_logits + (~att_mask) * (-1e9)
```

or similar approach using `masked_fill()`.

I noticed the same issue in other implementations within the repo, e.g. [here](https://github.com/s3prl/s3prl/blob/ec8064b5889f81ca460fbe2c094ce576a6f120b7/s3prl/downstream/sv_voxceleb1/model.py#L166).

(edited to fix line 122 to line 123)
